modules/coreos-layering-configuring-on-remove.adoc

```diff
@@ -10,4 +10,4 @@ To prevent the custom layered images from taking up excessive space in your regi
 
 The credentials provided by the registry push secret that you added to the `MachineOSBuild` object must grant the permission for deleting an image from the registry. If the delete permission is not provided, the image is not removed when you delete the `MachineOSBuild` object.
 
-Note that the custom layered image is not deleted if the image is either currently in use on a node or is desired by the nodes, as indicated by the `machineconfiguration.openshift.io/currentConfig` or `machineconfiguration.openshift.io/desiredConfig` annotation on the node.
+Note that the custom layered image is not deleted if the image is either currently in use on a node or is desired by the nodes, as indicated by the `machineconfiguration.openshift.io/currentImage` or `machineconfiguration.openshift.io/desiredImage` annotation on the node, which are added to the node upon creating the `MachineOSConfig` object.
```

Comment:
```suggestion
The custom layered image is not deleted when the image is either currently in use on a node or is desired by the nodes, as indicated by the `machineconfiguration.openshift.io/currentImage` or `machineconfiguration.openshift.io/desiredImage` annotations on the node. Both annotations can be added to the node when you create the `MachineOSConfig` object.
```
clearer, I think? not certain if the object is created by the user or is an automatic process, but you get the gist of what I am suggesting....